lib/modules/manager/bazel-module/maven.ts

```diff
@@ -0,0 +1,111 @@
+import { z } from 'zod';
+import { MavenDatasource } from '../../datasource/maven';
+import { id as versioning } from '../../versioning/gradle';
+import type { PackageDependency } from '../types';
+import {
+  RecordFragmentSchema,
+  StringArrayFragmentSchema,
+  StringFragmentSchema,
+} from './fragments';
+
+const artifactMethod = 'artifact';
+const installMethod = 'install';
+const commonDepType = 'maven_install';
+export const mavenVariableRegex = /^maven.*/;
```

Comment:
For this and other regex usages, please:

```suggestion
export const mavenVariableRegex = regEx(/^maven.*/);
```